src/Jupyter/Visualization/DisplayableExceptionEncoders.cs

```diff
@@ -35,10 +46,10 @@ internal static string ToFriendlyName(this ICallable callable)
             : fullName;
         }
 
-        internal static string ToSourceLink(this StackFrame frame) =>
```

Comment:
Should this be an extension method or could it be moved to the class?

Reply:
Good point, no reason for the extension class anymore